filesystem/etc/calico/confd/templates/bird.cfg.template

```diff
@@ -59,43 +59,116 @@ template bgp bgp_template {
   graceful restart;  # See comment in kernel section about graceful restart.
 }
 
-# ------------- Node-to-node mesh -------------
-{{if (json (getv "/global/node_mesh")).enabled}}
-{{range $host := lsdir "/host"}}
-{{$onode_as_key := printf "/host/%s/as_num" .}}
-{{$onode_ip_key := printf "/host/%s/ip_addr_v4" .}}{{if exists $onode_ip_key}}{{$onode_ip := getv $onode_ip_key}}
-{{$nums := split $onode_ip "."}}{{$id := join $nums "_"}}
+# -------------------------------------------------
+# ------------- Node-to-node mesh -----------------
+{{- $node_cid_key := printf "/host/%s/cluster_id" (getenv "NODENAME")}}{{if exists $node_cid_key}}
```

Comment:
Yes, I agree... I've tried to use indentation to make it a bit more clear (rather than just a big block). 

I'll see if there's any other formatting magic (or comments) I can add to the code to make it more clear what's going on.